go/cmd/dolt/cli/documentation_helper.go

```diff
@@ -0,0 +1,265 @@
+// Copyright 2020 Liquidata, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"text/template"
+
+	"github.com/liquidata-inc/dolt/go/libraries/utils/argparser"
+)
+
+type CommandDocumentForMarkdown struct {
```

Comment:
Comment on exported function.

https://github.com/golang/go/wiki/CodeReviewComments#doc-comments

In the format:

https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences